include/graphblas/algorithms/matrix_factory.hpp

```diff
@@ -227,6 +232,14 @@ namespace grb::algorithms {
 			 *
 			 * The \a V_iter and \a V_end iterator pair must contain exactly the number
 			 * of elements that should appear on the requested diagonal.
+			 *
+			 * \warning In parallel I/O mode, the given iterator pair must match in
+			 *          terms of both contents and order of returned elements, to those
+			 *          of the row- and column-indices returned by the \em internally-
+			 *          defined iterators, which (currently) are given by:
+			 *            - #grb::utils::iterators::Range .
```

Comment:
extra dot ?

Reply:
```suggestion
			 *            - #grb::utils::iterators::Range
```

Reply:
The dot should be there (the sentence continues from the preceding line), but the space is not necessary (tested with doxygen just now). However, iterators->containers. I'll make both changes.